src/main/java/server/entities/Booking.java

```diff
@@ -9,65 +9,57 @@
     private int bookingId;
     private int clientId; //TODO check if client it is int or string
     private String facilityName;
-    private LocalDateTime start;
-    private LocalDateTime end;
+    private String day;
+    private LocalTime start;
+    private LocalTime end;
 
 
-    public Booking(int bookingId, int clientId, String facilityName, LocalDateTime start, LocalDateTime end) {
+    public Booking(String day, int bookingId, String facilityName, LocalTime start, LocalTime end) {
+        this.day = day;
         this.bookingId = bookingId;
-        this.clientId = clientId;
```

Comment:
not sure if it's important, but not storing clientID in booking means any client can edit/delete any booking by any client. I.e. do we want to validate client before allowing offset/extend/delete requests?

Reply:
i see, i think can disucss this later. Bcos validating by IP and port will mean that a client has to use the same machine when he wants to access the services, so im not sure if we can use IP and port as client Id, or if not what better ways are there to validate.

Reply:
okay, we can discuss. i think validation might not be an important thing to do for the scope of this project bah haha. but i was thinking IP+port represents userID or like session token in an actual scenario